### PR TITLE
Adjust recipes to add compatibility with vlp_v3.0.6 and latest commit of meta-mistylwb5p

### PR DESCRIPTION
**Pull request by matinlotfali**
This package doesn't exist in vlp_v3.0.6 and its presence is unimportant. So I remove it from the image.

### CODE REVIEW

File: recipes-core/images/mistysom-image.bbappend [hunk 1]
```diff
@@ -49,7 +48,7 @@ IMAGE_INSTALL_append = " wpa-supplicant"
 
 # WiFi Module Laird Sterling-LWB5+ (Disable other WiFi modules because this module compiles a different cfg80211.ko)
 IMAGE_INSTALL_append = " lwb5plus-sdio-div-firmware"
-IMAGE_INSTALL_append = " kernel-module-lwb5p-backports-laird"
+IMAGE_INSTALL_append = " kernel-module-lwb5p-backports-summit"
```
Comment (mistyron) [inline recipes-core/images/mistysom-image.bbappend:51]: @matinlotfali were you able to get the Laird module working, eventually? What's the story here?

**Review by mistyron — COMMENTED** [review]
please add a comment that explains the current state of the Laird module

File: recipes-core/images/mistysom-image.bbappend (hunk 1 above)
Comment (matinlotfali) [reply recipes-core/images/mistysom-image.bbappend]: Just fixed it!

Comment (matinlotfali) [reply recipes-core/images/mistysom-image.bbappend:51]: Yes I did. I will write details.

**Comment by matinlotfali** [comment]
This meta-mistysom PR is working on RZG2L with VLP 3.0.0 here: https://github.com/MistySOM/rzg2l/pull/63
And RZV2L with VLP 3.0.6 here: https://github.com/MistySOM/rzv2l/pull/81
It wouldn't pass the yocto build because the pipeline is using the master branch of RZG2L which doesn't include the latest laird submodule.

**Pull request merged**